src/app.py

```diff
@@ -19,50 +20,70 @@ def main():
     )
 
     # Designing the interface
-    st.title("Wildfire image crowdsourcing")
-    st.write("The image you upload here will be used to create a public image dataset for wildfire detection.")
+    #st.title("Collecte de photos de feux de forêt")
+    st.title("Améliorez la detection des départs de feux")
+    st.write("La photo que vous partagez ici sera utilisée pour créer un jeu de données public "
+             "pour la detection de feux de forêts.")
+    st.write("Idéalement partagez nous une photo depuis une vue assez dégagée sur une zone forestière, avec et/ou sans flammes/fumées")
+    #st.write("Lorsque vous partagez une photo de départ de feux, n'hésitez pas à "
+    #        "également partager une photo du même point de vue sans fumées ou flammes.")
```

Comment:
Are the commented sections here on purpose?

Reply:
Well, which wording do you prefer ? the one in comments or the one in line 27 ? I will delete comments/l27 then